src/components/common/Card/EntryCardCollection.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import EntryCard from "./EntryCard.js";
+import { isPropertySignature } from "typescript";
+
+export default function EntryCardCollection(props) {
+  return (
+    <div className="collection collection--small-cards">
+      {props.data.map((entry, i) => {
+        return <EntryCard entry={entry} key={i} />;
+      })}
+    </div>
+  );
+}
```

Comment:
It looks like this file may not be used?

Reply:
Yes, duplicate. Forgot to remove this folder.